.github/workflows/bump-version.yml

```diff
@@ -23,11 +41,19 @@ jobs:
     - run: |
         set -euxo pipefail
 
+        if [ "$VERSION" = latest ]; then
+          VERSION="$(curl -sL https://pypi.org/pypi/$PACKAGE/json | jq -r .info.version)"
```

Comment:
@asottile-sentry Does this need to pull from the sentry pypi somehow?

Reply:
it probably should, though we currently delete the json endpoints since they were unused

Reply:
let me know if we should followup on this